### DISCUSSION TITLE
Better Jail

### DISCUSSION THREAD

**Discussion by Italoplcl** [Zrips/CMI#4599] [Feature Requests]
**Description of feature/improvement request:**

---

send players to a jail where they must work in order to get out. Mine x number of blocks.

/ jail Dentra 1h / 1000b (blocks) Jail Cell1

like https://www.spigotmc.org/resources/blocksjail.73964/

**Comment by TheExceed** [comment]
That would be interesting. Like rank up requirements but for jails.

**Comment by mrfloris** [comment]
I wonder when we put someone in jail we can run a command to set their blocks mined under their user_meta jailmined, and then use the equation placeholder to see if the value is bigger than 1000 from current vs what's in user meta, and if so, run the command to unjail. 

https://www.zrips.net/cmi/commands/user-meta/
https://www.zrips.net/cmi/placeholders/

**Reply by Italoplcl** [reply]
I tried to do it but nothing good turned out for me

**Comment by Italoplcl** [comment]
seems good! a "small" job to lower jail time.
